editor-blender/core/actions/state/control_editor.py

```diff
@@ -88,6 +93,7 @@ async def add_control_frame():
     set_requesting(False)
 
 
+# TODO finish this function
```

Comment:
You should remove those TODOs after finishing them, or they're going to accumulate in our repo.